meilisearch-http/tests/index/get_index.rs

```diff
@@ -77,13 +77,12 @@ async fn get_and_paginate_indexes() {
     const NB_INDEXES: usize = 50;
     for i in 0..NB_INDEXES {
         server.index(&format!("test_{i:02}")).create(None).await;
+        server
+            .index(&format!("test_{i:02}"))
+            .wait_task(i as u64)
+            .await;
```

Comment:
any idea why this works?

Reply:
I'm not sure, but, I think the Windows instances have lower performances than the Linux ones. This increases the chances for `wait_task` to timeout:
<img width="310" alt="Capture d’écran 2022-06-07 à 16 14 07" src="https://user-images.githubusercontent.com/6482087/172403062-5f362d67-e557-4f6e-844f-a46b69274b89.png">

Reply:
windows is really not fun, it's going to slow down everything a little bit more :weary: